workflows/tasks/macs.wdl

```diff
@@ -53,23 +52,20 @@ task macs {
                 -n ~{output_name}
         fi
 
-        mv ~{output_name}_MACS_wiggle/treat/~{output_name}_treat_afterfiting_all.wig.gz ~{coverage_location}
-        
+        mv ~{output_name}_MACS_wiggle/treat/~{output_name}_treat_afterfiting_all.wig.gz ~{
+            coverage_location}
+
         mv ~{output_name}* ~{default_location}
 
         #move coverage files to new location
-        #mv ~{default_location}/~{output_name}_MACS_wiggle/treat/~{output_name}_treat_afterfiting_all.wig.gz ~{coverage_location}
+        #mv ~{default_location}/~{output_name}_MACS_wiggle/treat/~{output_name}_treat_afterfiting_all.wig.gz ~{
+            coverage_location}
```

Comment:
@a-frantz - This is a new one, but it seems like a bad line break. I'm assuming it didn't check that the line was a comment when it attempted to split on the expression.

Reply:
WDL doesn't have any concept of embedded comments, so this is expected. We can't know (with reasonable certainty) the interpolation is within a comment, so even "commented out" interpolations are processed and substitutions occur. So formatting proceeds as normal for these cases